packages/fs-detectors/src/isError.ts

```diff
@@ -0,0 +1,26 @@
+// These functions are copied from vercel/cli
```

Comment:
I'd love to move these functions to some shared directory within the monorepo. Then they can be tested and used reliably by any package within the repo. Can be a private package too

Reply:
Maybe we make a `@vercel/error` package, so it can be used elsewhere as well?